### PR TITLE
fix(tabs): move tabs style to theme

### PR DESCRIPTION
**Pull request by mtikhinskaya**
**What is the problem this PR is trying to solve?**

theme tabs style differ from TabBar component style

**What is the chosen solution to this problem?**

TabBar component style moved to theme nav-tabs

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/theme/src/theme/_navs.scss [hunk 1]
```diff
@@ -73,41 +73,42 @@
 	}
 
 	&-tabs {
+		background-color: tint($brand-primary, 90);
```
Comment (frassinier) [inline packages/theme/src/theme/_navs.scss]: Is there a way to create some global values for these tint in variables.scss?
Like https://github.com/Talend/ui/blob/master/packages/theme/src/theme/_variables.scss#L21

Comment (mtikhinskaya) [inline packages/theme/src/theme/_navs.scss]: I think it should be a separate PR . We need to search occurrences and create a variable if it really needed. And not only for this color but for other ones

Comment (jsomsanith) [inline packages/theme/src/theme/_navs.scss]: This means we need 2 new shades of $brand-primary : 
* tab background
* tab active background
We already have $brand-primary-light, $brand-primary-lighter. If we come up with 2 new names that express that they are more light, ok for me.

Comment (mtikhinskaya) [inline packages/theme/src/theme/_navs.scss]: I don't understand what for we need this variable. It's used only in one place. It shouldn't be rewritten. It's not rewrite bootstrap variable. Could you please explain?

**Comment by jsomsanith** [comment]
Close for no common solution and non activity (means this is not needed)
